src/main/java/org/spongepowered/api/world/server/ChunkManager.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.world.server;
+
+import org.spongepowered.api.util.Ticks;
+import org.spongepowered.api.world.chunk.Chunk;
+import org.spongepowered.math.vector.Vector3i;
+
+import java.util.Optional;
+
+/**
+ * Manages {@link Chunk chunks} for a {@link ServerWorld}.
+ */
+public interface ChunkManager {
+
+    /**
+     * Checks if the provided {@link Ticket} is valid for the world this manager
+     * represents.
+     *
+     * @param ticket The ticket to check.
+     * @return true if so
+     */
+    boolean isValid(Ticket<?> ticket);
+
+    /**
+     * Gets the default lifetime, in {@link Ticks}, of any ticket requested via
+     * this ticket manager.
+     */
+    Ticks getDefaultTicketLifetime(TicketType<?> ticketType);
+
+    /**
+     * Gets the {@link Ticks} remaining on the supplied ticket.
+     *
+     * @return The {@link Ticks}
+     */
+    Ticks getTimeLeft(Ticket<?> ticket);
+
+    /**
+     * Request a {@link Ticket} for the given {@link TicketType}.
+     *
+     * @param type The type of ticket to request.
+     * @param value The value to register the ticket with.
+     * @param <T> The type of the supplied {@code value}.
+     * @return The ticket, if granted.
+     */
+    <T> Optional<Ticket<T>> requestTicket(TicketType<T> type, Vector3i chunkPosition, T value);
```

Comment:
Radius?

Reply:
Additional, the chunkPosition really is the chunkOrigin that the radius operates off of.